models/orogen/skid4_control.rb

```diff
@@ -1,17 +1,19 @@
+require 'models/services/motion2d_control_loop'
+require 'models/services/joints_control_loop'
 require 'models/blueprints/control'
 
 class OroGen::Skid4Control::SimpleController
-    provides Base::ActuatorControllerSrv, :as => 'controller'
-    provides Base::Motion2DControlledSystemSrv, :as => 'controlled_system'
+    provides Rock::Services::JointsOpenLoopController, :as => 'controller'
```

Comment:
Currently, bundles/rock contains basic models for all the Rock components. That includes controldev and skid4_control. This works so far because orogen files are loaded only if the corresponding componet is there.

It seems anyways that controldev is not required anymore here (Actuator and Motion2D are part of base, and in any case a file in models/orogen should never call using_task_library for another)

The problem is that (1) I don't have the time and energy to convert all the models to the new syntax and (2) even though I did a lot of work to try to keep backward compatibility, it seems that I broke it.